vector/src/main/res/layout/adapter_item_room_view.xml

```diff
@@ -9,36 +9,41 @@
     <View
         android:id="@+id/indicator_unread_message"
         android:layout_width="8dp"
-        android:layout_marginRight="8dp"
         android:layout_height="match_parent"
+        android:layout_marginRight="8dp"
         android:background="@android:color/darker_gray" />
 
-    <im.vector.view.VectorCircularImageView
-        android:id="@+id/room_avatar"
-        android:layout_toRightOf="@+id/indicator_unread_message"
-        android:layout_width="@dimen/chat_avatar_size"
-        android:layout_height="@dimen/chat_avatar_size"
+    <RelativeLayout
+        android:id="@+id/room_avatar_layout"
```

Comment:
We'll have a FrameLayout inside a RelativeLayout, maybe it could be simplified to remove one level ?